src/shared/events/implementations/aws/index.js

```diff
@@ -1,9 +1,49 @@
-function enqueue(topic, payload) {
-    console.log(`aws enqueue: ${topic} ${JSON.stringify(payload, null, 4)}`);
+const AWS = require('aws-sdk');
+
+const sqs = new AWS.SQS();
+const { AWS_REGION, AWS_ACCOUNT_ID } = process.env;
+
+async function enqueue(topic, payload) {
+    const QueueUrl = `https://sqs.${AWS_REGION}.amazonaws.com/${AWS_ACCOUNT_ID}/${topic}`;
+
+    const params = {
+        MessageBody: JSON.stringify(payload),
+        QueueUrl: QueueUrl,
+    };
+
+    try {
+        const result = await sqs.sendMessage(params);
+
+        return result;
+    }
+    catch (err) {
+        console.error(err);
+        throw err;
+    }
 }
 
-function dequeue(topic) {
-    console.log(`aws dequeue: ${topic}`);
+async function dequeue(topic, MaxNumberOfMessages = 10) {
+    const QueueUrl = `https://sqs.${AWS_REGION}.amazonaws.com/${AWS_ACCOUNT_ID}/${topic}`;
+
+    const params = {
+        QueueUrl,
+        MaxNumberOfMessages,
+    };
+
+    try {
+        const result = await sqs.receiveMessage(params);
+
+        result.Messages = result.Messages.map(_ => ({
+            ..._,
+            Body: JSON.parse(_.Body),
+        }));
```

Comment:
```suggestion
        result.Messages = result.Messages.map(message => ({
            ...message,
            Body: JSON.parse(message.Body),
        }));
```

**_** birçok dilde "dismiss" yani değişkenin kullanılmayacağını ifade eden bir placeholder gibi kullanılıyor. burada kodu okuyan kişi için karmaşa oluşturabilir.